frontend/device/remarkable/device.lua

```diff
@@ -118,5 +153,17 @@ function Remarkable:reboot()
     os.execute("systemctl reboot")
 end
 
-return Remarkable
+local f = io.open("/sys/devices/soc0/machine")
+if not f then error("missing sysfs entry for a remarkable") end
 
+local deviceType = f:read("*all") 
+if deviceType == "reMarkable 2.0\n" then
+    logger.info("rm2 ", deviceType)
+    if not os.getenv("RM2FB_SHIM") then
+        error("reMarkable2 requires RM2FB to work")
```

Comment:
I think you can feel free to link to the repo here, that'll make for a slightly more user-friendly experience ;).